rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -77,6 +79,12 @@ vx_enum interpret_tensor_data_type(RocalTensorDataType data_type) {
             return VX_TYPE_FLOAT16;
         case RocalTensorDataType::UINT8:
             return VX_TYPE_UINT8;
+        case RocalTensorDataType::INT8:
```

Comment:
Are you adding these types here to support audio of all dtypes? If so, we need to make sure pybind supports tensors of these types first

Reply:
These are added for future use